packages/storybook-vue/src/stories/Select/SelectStates.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Select } from "@utrecht/component-library-vue";
+import { argTypes, options } from "./argTypes.ts";
+
+<Meta id="vue-select" title="Vue.js Component/Select/States" component={Select} argTypes={argTypes} />
+
+# Select States
+
+export const Template = (args) => ({
+  components: { Select },
+  setup() {
+    return { args };
+  },
+  template: `
+  <Select v-bind="args" @change="action" />
```

Comment:
I was wondering if we should make the `update:modelValue` event also show up in the Actions panel, to demonstrate the form bindings. What do you think?